docs(content-releases): add missing endpoints to backend docs (#19302)
strapi/strapi: docs/docs/docs/01-core/content-releases/01-backend.md

  }
  ```

**Get all releases with/without an entry**:

- method: `GET`
- endpoint: `/content-releases/`
- params:
  ```ts
  {
    contentTypeUid: string;

>     entryId: number;

Reviewer: Will entryId still be correct after your refactor for single-types? Aren't we moving to something like entryUid or entrySlug?

```
packages/core/content-releases/server/src/services/validation.ts
```

## Migrations

We have two migrations that we run every time we sync the content types.

### `deleteActionsOnDisableDraftAndPublish`

> Every time an user disabled Draft and Publish in one Content Type we make sure to remove all the release actions related to entries of that content type to avoid errors.

Reviewer: A user, it's an exception to the rule 🤦‍♂️ 
```suggestion
When a user disables Draft and Publish in one Content Type we make sure to remove all the release actions related to entries of that content type to avoid errors.
```

I think when disabling draft and publish all _drafts_ are deleted. This makes me think should probably not delete _all_ entries of that contentType but rather just the drafts 😬

### `deleteActionsOnDeleteContentType`

> Similar to last one, we make sure to delete all the release actions related to entries from a just deleted Content Type.

Reviewer: ```suggestion
When a Content Type is deleted, delete all actions containing entries from that Content Type.
```

## Subscribing to Lifecycles Events

> We also make sure to subscribe the delete events on the database. So we can remove actions with entries just deleted.

Reviewer: ```suggestion
When an entry is deleted delete all actions containing that entry.
```
